espnet2/asr/state_spaces/utils.py

```diff
@@ -4,8 +4,15 @@
 import functools
 from typing import Callable, Mapping, Sequence
 
-import hydra
-from omegaconf import DictConfig, ListConfig
+try:
+    import hydra
+    from omegaconf import DictConfig, ListConfig
+except ImportError:
+    import logging
+
+    logging.warning(
+        "If you are running state-space model," "run `pip install espnet['task-asr']`."
+    )
```

Comment:
Is it correct?
`hydra` and `omegaconf` are related to `espnet-3`.
Why do they appear here?
What we need to install `task-asr`

Reply:
It looks like the files under `espnet2/asr/state_spaces/` use Hydra and OmegaConf.
Also, `opt_einsum` is only used within this directory, so I’ve included these in the `task-asr` group.